NitroxServer/Communication/Packets/Processors/MultiplayerSessionReservationRequestProcessor.cs

```diff
@@ -29,7 +29,7 @@ public override void Process(MultiplayerSessionReservationRequest packet, Nitrox
                 authenticationContext,
                 correlationId);
 
-            Log.Info($"Reservation processed successfully: Username: {packet.AuthenticationContext.Username} - {reservation}");
+            Log.Info($"Reservation processed successfully: Username: {packet.AuthenticationContext.Username} - {reservation} - IP Address: {connection.Endpoint.Address.ToString()}");
```

Comment:
We don't want to write the IP address to the log files for security reasons, so if you think logging the address is needed here use "InfoSensitive" and refer to examples in our code base for implementation.